### ISSUE TITLE
Running/extending BCILAB with GNU Octave

### ISSUE THREAD

**Issue by brylie** [sccn/BCILAB#36]
For users and contributors who don't use MATLAB, how might we run and extend the BCILAB? E.g. what are some opportunities and/or difficulties running BCILAB with GNU Octave? What are the possibilities of porting BCILAB to a fully open-source platform?

**Comment by brylie** [comment]
I have read the article [EEGLAB and supercomputing applications using free alternatives to Matlab](https://sccn.ucsd.edu/wiki/EEGLAB_and_Octave).

Are there any further efforts to port the user interface components to a cross-platform framework, such as Qt?

It just seems like BCILAB could reach a much wider audience if it were decoupled a bit more from MATLAB.

**Comment by PeterGamma** [comment]
I agree with brylie completely. Availablity of BCILAB for Octave would allow to use it together with EEGLAB for mobile use on Linux Phones like the Linux PinePhone or the Xiaomi Pocophone F1 running Ubuntu Touch. These devices are affordable and very attractive for mobile EEG data aquisiton.

**Comment by arnodelorme** [comment]
I do not think that it is too far out if somebody wanted to take the lead on that.
